test/unit/cms/file_test.rb

```diff
@@ -39,6 +39,9 @@ class CMS::FileTest < ActiveSupport::TestCase
     file.attachment = Rails.root.join('test', 'fixtures', 'test.css').open
 
     assert_equal 'text/css', file.attachment_content_type
+
+    file.valid?
```

Comment:
```suggestion
    refute file.valid?`
```

Reply:
I don't mind doing this 👍 but this wasn't meant to be a validation originally... just called to get the errors 😄